plugins/ssm-secret.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ssm-secret
+spec:
+  version: v1.2.1
+  homepage: https://github.com/pr8kerl/kubectl-ssm-secret
+  caveats: |
+    Read the documentation at https://github.com/pr8kerl/kubectl-ssm-secret.
+    You can call this plugin with "kubectl ssm-secret".
+  shortDescription: import/export kubernetes secrets from/to AWS SSM parameter store paths.
```

Comment:
Please capitalize sentence.

Anything you can do to make this shorter is appreciated. For example, maybe remove `kubernetes`. How about:

> Import/export Secrets from/to AWS SSM store

Reply:
It's really better to be succinct here, because this will usually get truncated at ~50 chars.